Headers/DebugServer2/GDBRemote/Session.h

```diff
@@ -143,8 +143,8 @@ class Session : public SessionBase {
   void Handle_qOffsets(ProtocolInterpreter::Handler const &,
                        std::string const &);
   void Handle_qP(ProtocolInterpreter::Handler const &, std::string const &);
-  void Handle_qPlatform_IO_MkDir(ProtocolInterpreter::Handler const &,
-                                 std::string const &);
+  void Handle_qPlatform_mkdir(ProtocolInterpreter::Handler const &,
```

Comment:
That makes sense. Thanks for checking that stuff.

On Wed, Jun 15, 2016 at 4:01 PM, Francis Ricci <notifications@github.com>
wrote:

> In Headers/DebugServer2/GDBRemote/Session.h
> <https://github.com/facebook/ds2/pull/375#discussion_r67261613>:
>
> > @@ -143,8 +143,8 @@ class Session : public SessionBase {
> >    void Handle_qOffsets(ProtocolInterpreter::Handler const &,
> >                         std::string const &);
> >    void Handle_qP(ProtocolInterpreter::Handler const &, std::string const &);
> > -  void Handle_qPlatform_IO_MkDir(ProtocolInterpreter::Handler const &,
> > -                                 std::string const &);
> > +  void Handle_qPlatform_mkdir(ProtocolInterpreter::Handler const &,
>
> It's not. I did a google search for the former, and the only results were
> some old lldb patches. A grep in the gdb and lldb sources for the former
> return nothing, and only the qPlatform_mkdir shows up in their docs. My
> assumption is that they started implementing and then decided it was a bad
> name.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/375/files/687555e04395f15d83108877ce62d7c6805a3aa9#r67261613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbHDbpDTj3KSA3CkV3lAA7nCj6zVxks5qMIRUgaJpZM4I229L>
> .
>



-- 
Stephane Sezer